invenio_records_files/api.py

```diff
@@ -85,10 +115,22 @@ def dumps(self):
 
 
 def _writable(method):
-    """Check that record is in defined status."""
+    """Check that record is in defined status.
+
+    :param method: Method to be decorated.
+    :returns: Function decorated.
+    """
     @wraps(method)
     def wrapper(self, *args, **kwargs):
-        """Send record for indexing."""
+        """Send record for indexing.
+
+        :param args: List of arguments.
```

Comment:
IMHO it is not necessary to document internal `*args`, `**kwargs`. It doesn't add any value.